ui/app/AppLayouts/Profile/stores/ProfileSectionStore.qml

```diff
@@ -154,6 +158,51 @@ QtObject {
         }
     }
 
+    readonly property alias ownShowcaseCommunitiesModel: ownShowcaseModels.adaptedCommunitiesSourceModel
+    readonly property alias ownShowcaseAccountsModel: ownShowcaseModels.adaptedAccountsSourceModel
+    readonly property alias ownShowcaseCollectiblesModel: ownShowcaseModels.adaptedCollectiblesSourceModel
+    readonly property alias ownShowcaseSocialLinksModel: ownShowcaseModels.adaptedSocialLinksSourceModel
+
+    readonly property alias contactShowcaseCommunitiesModel: contactShowcaseModels.adaptedCommunitiesSourceModel
+    readonly property alias contactShowcaseAccountsModel: contactShowcaseModels.adaptedAccountsSourceModel
+    readonly property alias contactShowcaseCollectiblesModel: contactShowcaseModels.adaptedCollectiblesSourceModel
+    readonly property alias contactShowcaseSocialLinksModel: contactShowcaseModels.adaptedSocialLinksSourceModel
+
+    function requestContactShowcase(address) {
+        root.contactsStore.requestProfileShowcase(address)
+    }
+
+    function requestOwnShowcase() {
+        root.profileStore.requestProfileShowcasePreferences()
+    }
+
+    readonly property QObject d: QObject {
```

Comment:
Just a cuestion @alexjba @micieslak for the future.. I'm not sure if we should instantiate the adapters inside the store.. :thinking: I don't have know a better alternative but, just something to think on!

Reply:
An alternative would be to instantiate them where it's being used. We could end up with multiple adapter instances